inspire_json_merger/merger_config.py

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from json_merger.config import DictMergerOps, UnifierOps
+from .comparators import COMPARATORS
+
+ARXIV_TO_ARXIV = 'ArxivToArxiv'
+PUBLISHER_TO_ARXIV = 'PublisherToArxiv'
+
+
+class MergerConfigurationOperations(object):
+    default_dict_merge_op = DictMergerOps.FALLBACK_KEEP_UPDATE
+    default_list_merge_op = UnifierOps.KEEP_ONLY_UPDATE_ENTITIES
+    list_dict_ops = None
+    list_merge_ops = None
+    comparators = None
+
+    def factory(type):
```

Comment:
You can add here the decorator: 
```
@staticmethod
```

But, why do you use a static method at all? I believe in Python there is no such requirement for implementing a factory.

See also: https://stackoverflow.com/a/15031999/1407497